drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/pom.xml

```diff
@@ -162,6 +162,30 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
I understand that it might seem confusing but you probably don't need both `uberfire-testing-utils` and `uberfire-test-utils`

[ubefire-test-utils](https://github.com/kiegroup/appformer/tree/master/uberfire-test-utils) is a module that was recently moved from guvnor's guvnor-test-utils and renamed. Looking at the sources it mainly contains utilities to make setup of CDI (weld-se -based) tests easier. In the long run I think it would be best to refactor that module a bit by:
1. Renaming it to *ubefire-backend-test-utils* to prevent confusion
2. Simplifying the weld setup a bit (along the lines of [this PR](https://github.com/kiegroup/kie-wb-common/pull/1307))


[uberfire-testing-utils](https://github.com/kiegroup/appformer/tree/master/uberfire-testing-utils) is a module that's been in uberfire for a long time and contains utility classes primarily for client-side testing with mockito (e.g. CallerMock etc). I think you don't need to add that as a test dependency in this case. In the long run I think this should be renamed to *uberfire-client-test-utils*

@ederign would you agree with renaming 
uberfire-test-utils -> uberfire-baskend-test-utils
uberfire-testing-utils -> uberfire-client-test-utils

Reply:
Great, understood, will preserve just the correct one.